src/ModelManager.ts

```diff
@@ -552,6 +571,34 @@ export class ModelManager {
 
         return Promise.resolve();
     }
+
+    /**
+     * Fetches parent page information of the given component path
+     * Returns object containing
+     * 1. Parent page path
+     * 2. Parent page data if already available in the store
+     * @return {object}
+     */
+    private _fetchParentPage(path: string): ParentPage {
```

Comment:
Is it fetching in terms of doing a call?

Reply:
@CezCz Retrieving from the ModelStore - is the naming confusing in this context?

Reply:
i am just wondering if type could be more generic than `ParentPage`? but if you are convinced it cannot be reused, i am fine with the name